queries.sql

```diff
@@ -157,4 +157,55 @@ SELECT
 FROM animals
 WHERE DATE_PART('YEAR', date_of_birth) >= 1990
   AND DATE_PART('YEAR', date_of_birth) <= 2000
-GROUP BY neutered;
\ No newline at end of file
+GROUP BY neutered;
+
+
+-- What animals belong to Melody Pond?
+SELECT name
+FROM animals
+INNER JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name
+FROM animals
+INNER JOIN species ON  animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT owners.full_name AS owner, STRING_AGG (animals.name, ', ') AS animals
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id
+GROUP BY owners.full_name;
+
+-- How many animals are there per species?
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/220857978-6f36d58d-46f3-4e96-b4ae-d8c0ee36f831.png)

- [ ] Kindly remove this section description to make your readme file descriptive and clean :blush:

**Section Description**

`Describe 1 - 3 features you will add to the project.`